apiserver/src/client/webclient.rs

```diff
@@ -119,8 +119,8 @@ impl K8SAPIServerClient {
     fn https_client() -> Result<reqwest::Client> {
         let mut buf = Vec::new();
 
-        let ca_path = format!("{}/{}", TLS_KEY_MOUNT_PATH, CA_NAME);
-        std::fs::File::open(ca_path)
+        let leaf_cert_path = format!("{}/{}", TLS_KEY_MOUNT_PATH, PUBLIC_KEY_NAME);
+        std::fs::File::open(leaf_cert_path)
```

Comment:
I'm learning new things reviewing this PR, so grain of salt,

but don't we still want this to be the `ca.crt` file, rather than the client's public key? My understanding is that the client is using this to validate that the server's public key is signed by our CA.

Reply:
True - that does make more sense! Made that change!